rrfs-test/CMakeLists.txt

```diff
@@ -94,11 +98,20 @@ if(FV3_DYCORE)
 	 file(COPY ${bl_FILES} DESTINATION ${casedir})
 	 file(COPY ${src_yaml}/${case}.yaml DESTINATION ${casedir})
          set(target_test ${case})
-         ecbuild_add_test( TARGET ${target_test}
+	 if ("${target_test}" STREQUAL "rrfs_mpasjedi_2024052700_getkf_solver")
+           ecbuild_add_test( TARGET ${target_test}
                         MPI      120
-                        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/rundir-${target_test}
+			WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/rundir-${target_test}
+                        ARGS     ${target_test}.yaml
+			TEST_DEPENDS "rrfs_mpasjedi_2024052700_getkf_observer"
+                        COMMAND  ${exe} )
+	else()
```

Comment:
It looks like the indentation of line 108 is not aligned with line 101

Reply:
Thanks for catching this. Should be resolved now.